### PR TITLE
Core/PacketIO: Update UpdateFlags for SMSG_UPDATE_OBJECT movement update

### PR DESCRIPTION
**Pull request by Artamedes**
* label some fields
* also rename SMSG_PLAYER_MOVE to SMSG_MOVE_UPDATE to match client name

### CODE REVIEW

File: src/server/game/Server/Protocol/Opcodes.cpp [hunk 1]
```diff
@@ -1052,6 +1052,7 @@ void OpcodeTable::Initialize()
     DEFINE_SERVER_OPCODE_HANDLER(SMSG_MOVE_UNSET_CAN_FLY,                               STATUS_NEVER,       CONNECTION_TYPE_INSTANCE);
     DEFINE_SERVER_OPCODE_HANDLER(SMSG_MOVE_UNSET_CAN_TRANSITION_BETWEEN_SWIM_AND_FLY,   STATUS_NEVER,       CONNECTION_TYPE_INSTANCE);
     DEFINE_SERVER_OPCODE_HANDLER(SMSG_MOVE_UNSET_HOVER,                                 STATUS_NEVER,       CONNECTION_TYPE_INSTANCE);
+    DEFINE_SERVER_OPCODE_HANDLER(SMSG_MOVE_UPDATE,                                      STATUS_NEVER,       CONNECTION_TYPE_REALM);
```
Comment (Ovahlord) [inline src/server/game/Server/Protocol/Opcodes.cpp]: connection_type_instance for movement opcodes

File: src/server/game/Entities/Object/Object.cpp [hunk 2]
```diff
@@ -467,8 +468,8 @@ void Object::BuildMovementUpdate(ByteBuffer* data, uint16 flags) const
 
     // Data
     if (GameObject const* go = ToGameObject())
-        for (uint32 i = 0; i < stopFrameCount; ++i)
-            *data << uint32(go->GetGOValue()->Transport.StopFrames->at(i));
+        if (stopFrameCount)
+            data->append(go->GetGOValue()->Transport.StopFrames->data(), go->GetGOValue()->Transport.StopFrames->size());
```
Comment (Warpten) [inline src/server/game/Entities/Object/Object.cpp]: This isn't exactly pretty and would break in the unlikely event that StopFrames becomes non-contiguous, please revert

File: src/server/game/Entities/Object/Updates/UpdateData.h [hunk 3]
```diff
@@ -40,17 +40,18 @@ enum OBJECT_UPDATE_FLAGS
...
-    UPDATEFLAG_LIVING                = 0x0020,
-    UPDATEFLAG_STATIONARY_POSITION   = 0x0040,
-    UPDATEFLAG_VEHICLE               = 0x0080,
-    UPDATEFLAG_GO_TRANSPORT_POSITION = 0x0100,
-    UPDATEFLAG_ROTATION              = 0x0200,
-    UPDATEFLAG_UNK3                  = 0x0400,
-    UPDATEFLAG_ANIMKITS              = 0x0800,
-    UPDATEFLAG_UNK5                  = 0x1000,
-    UPDATEFLAG_UNK6                  = 0x2000,
+    UPDATEFLAG_LOWGUID               = 0x0008,
```
Comment (Warpten) [inline src/server/game/Entities/Object/Updates/UpdateData.h]: Cant check these because I didnt bother naming values update blocks, but why did you rename/shuffle these while they were read by the client up until 4.3.0

Comment (Artamedes) [reply src/server/game/Entities/Object/Updates/UpdateData.h]: they are not sent to client anymore, serverside only

Comment (Warpten) [reply src/server/game/Entities/Object/Updates/UpdateData.h]: Sure, but I'd keep them in sync with old values, because changing them doesn't even matter. Also keeps in sync with WPP, otherwise cue confusion at some point in the future.
